docs/guide/http_cache.md

```diff
@@ -300,7 +305,9 @@ Somehow we need to tell php process that we are behind a Varnish proxy and not t
  
     # Force front controller NOT to use built-in reverse proxy.
     SetEnv SYMFONY_HTTP_CACHE 0
- 
+    SetEnv SYMFONY_HTTP_CACHE http
```

Comment:
this will work, but as of 2.1 with only has http cache via ezplatform.http-cache, it also has alias as `varnish` on top of the `fastly` one provided by fastly bundle.

So for a master PR I guess it should rather set value as varnish.

ref: https://github.com/ezsystems/ezplatform-http-cache/blob/master/src/Resources/config/services.yml#L13-L18